app/src/main/java/com/bootcamp_android/parking_app/fragments/LotListFragment.kt

```diff
@@ -12,11 +14,16 @@ import com.bootcamp_android.parking_app.databinding.FragmentLotsBinding
 class LotListFragment : Fragment(R.layout.fragment_lots) {
 
     override fun onViewCreated(itemView: View,savedInstanceState: Bundle?) {
+
         val binding = FragmentLotsBinding.bind(itemView)
         binding.recyclerLotList
```

Comment:
It's repeated

Reply:
do u mean white space 😬

Reply:
sorry i dont understand, what is the problem?